app/javascripts/components/playSlot/reducer.jsx

```diff
@@ -30,6 +32,47 @@ export const PLAY_SLOT_INITIAL_STATE = fromJS({
 
 export function reducer(state = PLAY_SLOT_INITIAL_STATE, action) {
   switch (action.type) {
+    case ACTION_TYPES.START_TO_GET_SLOT_MACHINE: {
+      return state.withMutations(currentState => {
+        return currentState.set('isLoading', true).set('hasError', false);
+      });
+    }
+
+    case ACTION_TYPES.FAILED_TO_GET_SLOT_MACHINE: {
+      return state.withMutations(currentState => {
+        return currentState.set('isLoading', false).set('hasError', true);
+      });
+    }
+
+    case ACTION_TYPES.SUCCEEDED_TO_GET_SLOT_MACHINE: {
+      return state.withMutations(currentState => {
+        return currentState
+          .set('isLoading', false)
+          .set('hasError', false)
+          .set('minBet', parseFloat(action.payload.minBet))
+          .set('betSize', parseFloat(action.payload.minBet))
+          .set('maxBet', parseFloat(action.payload.maxBet))
```

Comment:
suggestion)
You can round up to the specific digit after the decimal point.
I suggest 3 point.

```
parseFloat(action.payload.minBet).toFixed(3);
```